src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -148,12 +188,17 @@ const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
   ): TrezorOutput => {
     const changeAddress = getChangeAddress(changeOutputFiles, output.address, network)
     const address = encodeAddress(output.address)
+    const lovelaceAmount = `${output.coins}`
+    const tokenBundle = output.tokenBundle && prepareTokenBundle(output.tokenBundle)
```

Comment:
the `output.tokenBundle &&` part propably isnit nessesary

Reply:
It has to be somewhere or else it would throw error on threshold version check. I moved it inside `prepareTokenBundle` to be consistent with other `prepare...` functions.